data/users/externaloidc.py

```diff
@@ -37,10 +39,37 @@ def verify_credentials(self, username_or_email, password):
         """
         Unsupported to login via username/email and password
         """
-        return (
-            None,
-            f"Unsupported login option. Please sign in with the configured {self._federated_service} provider",
-        )
+        if not password:
+            return (None, "Anonymous binding not allowed.")
+
+        if not username_or_email:
+            return (None, "Missing username or email.")
+
+        for service in app.oauth_login.services:
```

Comment:
can there be multiple instances in this case? Can you identify the correct OIDC service and send the creds there? otherwise you'll be sending the full password to a service that's not supposed to receive it